Update list of plugins emoji legend for deuteranopia (#555)
remarkjs/remark: doc/plugins.md

remark.
Their status is encoded below as:

> *   🟢 This plugin **is not affected** and works the same as before,
> or this plugin is affected: **a new version is already released**

Reviewer: ```suggestion
*   🟢 This plugin **was not affected** or **a new version is already released**
```
